backends-velox/src/main/scala/org/apache/gluten/execution/ColumnarPartialProjectExec.scala

```diff
@@ -59,10 +59,8 @@ case class ColumnarPartialProjectExec(original: ProjectExec, child: SparkPlan)(
   private val projectAttributes: ListBuffer[Attribute] = ListBuffer()
   private val projectIndexInChild: ListBuffer[Int] = ListBuffer()
   private var UDFAttrNotExists = false
-  private var hasUnsupportedDataType = replacedAliasUdf.exists(a => !validateDataType(a.dataType))
```

Comment:
The check is duplidate within `getProjectIndexInChildOutput`, so removed one cc @jinchengchenghh